client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts

```diff
@@ -89,6 +89,10 @@ export class AgendaListComponent extends ListViewBaseComponent<ViewItem> impleme
         this.initTable();
         this.filterService.filter().subscribe(newAgendaItems => {
             newAgendaItems.sort((a, b) => a.agendaListWeight - b.agendaListWeight);
+
+            // remove those with empty content objects
+            newAgendaItems = newAgendaItems.filter(item => item.contentObject);
```

Comment:
Please do not do this. This is intentional, if someone can see the agenda, but not thier content objects, E.g. when can_see_motion=False. The agenda should still be complete for this user.

Reply:
That is the point.
The workflow option "only managers of motions" will get content objects.
If you do not have the content object of the agenda item, you are not supposed to see it in the agenda.

@emanuelschuetze

Reply:
The agenda is completly independed of the accesspermissions of content objects, it does not have any relationships. If so, this needs to be fixed in the server. E.g. set type "internal item" automatically.